### PR TITLE
[5.1] Fixed support for PHP 7.1

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
I'll look into that broken crap.

File: tests/Encryption/EncrypterTest.php [hunk 1]
```diff
@@ -86,6 +86,10 @@ public function testOpenSslEncrypterCanDecryptMcryptedData()
             $this->markTestSkipped('Mcrypt module not installed');
         }
 
+        if (version_compare(PHP_VERSION, '7.1') > 0) {
+            $this->markTestSkipped('Not compatable with PHP 7.1+');
+        }
```
Comment (taylorotwell) [inline tests/Encryption/EncrypterTest.php]: Why?

Comment (GrahamCampbell) [inline tests/Encryption/EncrypterTest.php]: You can still install the mcrypt extension on PHP 7.1 (infact, travis has), but running it fail the tests due to the deprecation.

Comment (vlakoff) [inline tests/Encryption/EncrypterTest.php]: typo "compatible" ;)

Comment (vlakoff) [inline tests/Encryption/EncrypterTest.php]: Also the comparison operator should be `>= 0`, or you can do ` if (version_compare(PHP_VERSION, '7.1', '>='))`

**Pull request merged**